Orange/tests/test_statistics.py

```diff
@@ -1,15 +1,17 @@
 import unittest
 import warnings
+from distutils.version import LooseVersion
 from itertools import chain
 from functools import partial, wraps
 
 import numpy as np
 from scipy.sparse import csr_matrix, issparse, lil_matrix, csc_matrix, \
     SparseEfficiencyWarning
 
+import Orange
 from Orange.statistics.util import bincount, countnans, contingency, digitize, \
     mean, nanmax, nanmean, nanmedian, nanmin, nansum, nanunique, stats, std, \
-    unique, var, nanstd, nanvar, nanmode, array_equal, FDR
+    unique, var, nanstd, nanvar, nanmode, FDR
 from sklearn.utils import check_random_state
```

Comment:
You mean because of the alphabetical order? It was in the wrong order before this PR, but I removed it, so it should be fine. Or did you mean something else?